.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
```

Comment:
What about: `check-added-large-files` too?

500 kb limit by default, which seems reasonable.

Reply:
:+1:

Reply:
Can you alphabetize this list so it is easier to parse?

Reply:
done